src/pagination/types.ts

```diff
@@ -5,6 +5,14 @@ export interface PaginationsProps {
     pageSize?: number | undefined;
     totalItems: number;
     activePage: number;
+    pageSizeOptions?: DropdownItemProps[];
     showFirstAndLastNav?: boolean | undefined;
+    showPageSizeChanger?: boolean | undefined;
     onPageChange?: ((page: number) => void) | undefined;
+    onPageSizeChange?: ((page: number, pageSize: number) => void) | undefined;
+}
+
+export interface DropdownItemProps {
```

Comment:
ok, I see it's now exposed as an option. in that case could you rename this to `PageSizeItemProps`